<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0,0,0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210,210,210);</string>
    </property>
    <widget class="QWidget" name="mainLoginPage">
     <widget class="QLabel" name="signTitleLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passworlLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="passwordForgotBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Забыли пароль?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signInBtn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainRegistPage">
     <widget class="QLabel" name="registTitleLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="registLoginLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="registPasswordLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="registRePasswordLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registLoginLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="registPasswordLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="registRePasswordLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createUserBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainForgotPasswordPage"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
